lib/setup.py

```diff
@@ -30,7 +30,7 @@
 # - And include an upper bound that's < NEXT_MAJOR_VERSION
 INSTALL_REQUIRES = [
     "altair>=4.0, <6",
-    "blinker>=1.0.0, <2",
+    "blinker>=1.5.0, <2",
```

Comment:
blinker pre 1.5 is very old and has been throwing deprecation warnings when used. I think it's about time to increase the lower pin.